Debonair-Squids/Bangazon_WebApp
# Fixed Add Product Method

# Steps To Test
1. git fetch --all
2. git checkout TB-Working
3. Check to see if you can add a product successfully.


### Description of what should happen when steps are followed
You should be able to successfully add a product to the database.

**At least two people needs to approve pull request before merged to master**
